### PR TITLE
Use custom package_paths in EPR

### PR DESCRIPTION
**Pull request by mtojek**
This PR uses freshly introduced `package_paths` configuration option to prevent from pulling packages from the package-storage.

### CODE REVIEW

File: testing/environments/package-registry.config.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+package_paths:
+  - /registry/packages/package-storage
```
Comment (ruflin) [inline testing/environments/package-registry.config.yml:2]: What is the current behaviour if the two contain the same version of a package?

Comment (mtojek) [reply testing/environments/package-registry.config.yml:2]: First found served. Probably need a fix for this in the registry. Also, there is an issue with updating content. If I have the EPR running and modify a `manifest.yml` (let's say a description), this change is not reflected.

Comment (mtojek) [reply testing/environments/package-registry.config.yml:2]: Here: https://github.com/elastic/package-registry/blob/master/util/packages.go#L16

We can either remove this global var or add a switch to always load packages from disk.

Comment (ruflin) [reply testing/environments/package-registry.config.yml:2]: As you probably guessed, I like the cache. But I also like the idea that for development purpose it does not use any. We could use a `-dev` flag or similar?

File: magefile.go [hunk 2]
```diff
@@ -22,21 +22,13 @@ import (
 )
 
 var (
-	// GoImportsImportPath controls the import path used to install goimports.
-	GoImportsImportPath = "golang.org/x/tools/cmd/goimports"
```
Comment (ruflin) [inline magefile.go:26]: I see you remove the licenser and imports part. Don't you need this for the golang files + headers?

Comment (mtojek) [reply magefile.go:26]: I noticed that these paths are not used at all. Maybe it's a good point to bring the logic back.

File: testing/environments/package-registry.config.yml (hunk 1 above)
Comment (mtojek) [reply testing/environments/package-registry.config.yml:2]: `-dev` would work. I will try to introduce it quickly to EPR.

**Comment by mtojek** [comment]
> If I follow this change correctly, it means to test with the package-storage it is required to test with Docker?

In the integrations, you focus on testing the integration, not burning CPU cycles on verifying the package-storage, hence no need to check it out.

File: testing/environments/package-registry.config.yml (hunk 1 above)
Comment (mtojek) [reply testing/environments/package-registry.config.yml:2]: https://github.com/elastic/package-registry/pull/543

File: magefile.go (hunk 2 above)
Comment (ruflin) [reply magefile.go:26]: Yes, these are important to add license headers etc.

**Comment by ruflin** [comment]
@mtojek I'm good with just having just integrations as part of the build. I think it the past we needed the storage because `system`, `base` packages were missing.

File: magefile.go (hunk 2 above)
Comment (mtojek) [reply magefile.go:26]: The code for license verification is present:

```golang
// AddLicenseHeaders adds license headers to .go files. It applies the
// appropriate license header based on the value of mage.BeatLicense.
func AddLicenseHeaders() error {
	fmt.Println(">> fmt - go-licenser: Adding missing headers")
	return sh.RunV("go-licenser", "-license", "Elastic")
}
```

**Pull request merged**